RevenueCatUI/Templates/V2/Components/Video/VideoComponentView.swift

```diff
@@ -161,12 +148,13 @@ struct VideoComponentView: View {
                         stagedURL.publisher
                             .eraseToAnyPublisher()
                             .removeDuplicates()
-                            .debounce(for: 0.300, scheduler: RunLoop.main)
```

Comment:
This worked well in the original context, but not so well in backgrounds 🤷‍♂️  So I changed it